### PR TITLE
Making accessibility changes to Pdf View in Reader

### PR DESCRIPTION
**Pull request by tejans24**
connects #1853

### CODE REVIEW

File: client/app/components/Accordion.jsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default class Accordion extends React.PureComponent {
           disabled: child.props.disabled
         })}
         key={child.props.title}
-        header={child.props.title}>
+        header={<h3>{child.props.title}</h3>}>
```
Comment (tejans24) [inline client/app/components/Accordion.jsx]: I think I prefer passing header as a (H3) Node. Incase this needs to be customized else where with a different header tag. But I'm also ok holding off until we need to.

File: app/assets/stylesheets/_reviewer.scss [hunk 2]
```diff
@@ -145,6 +145,13 @@ svg[aria-hidden=true],
   .usa-alert {
     margin-top: 15px;
   }
+
+  [id="hide-menu-header"] {
```
Comment (mdbenjam) [inline app/assets/stylesheets/_reviewer.scss]: Can we consolidate these into one class that's something like

```
.cf-non-stylized-header {
  font-size: inherit;
  margin: 0;
  color: inherit;
}
```

File: app/assets/stylesheets/_reviewer.scss [hunk 3]
```diff
@@ -591,6 +597,22 @@ svg[aria-hidden=true],
   }
 }
 
+[id="cf-pdf-header-doc-type"] {
```
Comment (mdbenjam) [inline app/assets/stylesheets/_reviewer.scss]: Can't you replace these two with the non-stylized-header class now?

File: app/assets/stylesheets/_reviewer.scss [hunk 4]
```diff
@@ -591,6 +598,12 @@ svg[aria-hidden=true],
   }
 }
 
+.usa-accordion-button {
```
Comment (mdbenjam) [inline app/assets/stylesheets/_reviewer.scss]: I think we should just use the same non stylized class.

Comment (tejans24) [inline app/assets/stylesheets/_reviewer.scss]: I was just thinking about doing that but it needed to be in the Accordion.jsx, I didn't want to add additional classes to the component. I'll make the change.

**Pull request merged**